src/main/java/org/spdx/sbom/gradle/utils/SpdxDocumentBuilder.java

```diff
@@ -350,6 +357,22 @@ private Optional<SpdxPackage> createMavenModulePackage(
     return Optional.empty();
   }
 
+  private String buildMavenPackageSupplier(PomInfo pomInfo) {
+    var organizationName = pomInfo.getOrganization().flatMap(o ->
+        Optional.ofNullable(o.getName())).map(n ->
+        "Organization: " + n);
+
+    var developer = pomInfo.getDevelopers().stream().filter(d -> d.getName().isPresent()).findFirst();
+    Optional<String> developerName = Stream.of(
+            developer.flatMap(d -> d.getName().map(name -> "Person: " + name)),
+            developer.flatMap(d -> d.getEmail().map(email -> " (" + email + ")"))
+        ).filter(Optional::isPresent)
+        .map(Optional::get)
+        .reduce((name, email) -> name + email);
+
```

Comment:
Do we want to support the possibility of a `developer` that has an `organization`?

I think an example is:

https://repo1.maven.org/maven2/org/jetbrains/annotations/13.0/annotations-13.0.pom

which contains the text:

```
<developers>
<developer>
<id>JetBrains</id>
<name>JetBrains Team</name>
<organization>JetBrains</organization>
<organizationUrl>http://www.jetbrains.com</organizationUrl>
</developer>
</developers>
```

which is think is causing this text to appear in the sbom:

“supplier" : "Person: Kotlin Team"

maybe it would be better to output this text instead:

“supplier" : "Organization: JetBrains"

Reply:
If a developer is listed as having an organization in the .pom, I wonder whether it's safe to assume that the developer's organization was also the organization that supplied the artifact. My guess would be probably?

Reply:
I would think that if the developer specified an organization it would be to differentiate between their organization and the organization that is supplying the artifact. 

The use case that comes to mind is a project developed by a foundation, where each developer wants to indicate what organization they're a part of (e.g. Google, Microsoft, etc...) but the supplying organization is the foundation (e.g. Apache).

Reply:
I'm actually seeing this pattern in google repos as well, using the developer field with only organization.